src/testers.jl

```diff
@@ -59,6 +77,35 @@ function _make_fdm_call(fdm, f, ȳ, xs, ignores)
     return (args...,)
 end
 
+"""
+    _make_jvp_call(fdm, f, xs, ẋs, ignores)
+
+Call `FiniteDifferences.jvp`, with the option to ignore certain `xs`.
+
+# Arguments
+- `fdm::FiniteDifferenceMethod`: How to numerically differentiate `f`.
+- `f`: The function to differentiate.
+- `xs`: Inputs to `f`, such that `y = f(xs...)`.
+- `ẋs`: Derivative of `xs` w.r.t. some real number `t`.
+- `ignores`: Collection of `Bool`s, the same length as `xs` and `ẋs`.
+   If `ignores[i] === true`, then `ẋs[i]` is ignored for derivative estimation.
+
+# Returns
+- `∂Ω`: Derivative of output w.r.t. `t` estimated by finite differencing.
+"""
+function _make_jvp_call(fdm, f, xs, ẋs, ignores)
+    f2 = _wrap_function(f, xs, ignores)
+
+    ignores = collect(ignores)
+    args = Any[nothing for _ in 1:length(xs)]
+    all(ignores) && return (args...,)
+    sigargs = zip(xs[.!ignores], ẋs[.!ignores])
+    arginds = (1:length(xs))[.!ignores]
+    fd = jvp(fdm, f2, sigargs...)
+
+    return fd
```

Comment:
Is it worth naming the variable here, or would it be just as clear to return it directly
```suggestion
    return jvp(fdm, f2, sigargs...)
```